src/test/suite/language/mongoDBService.test.ts

```diff
@@ -1246,4 +1247,160 @@ suite('MongoDBService Test Suite', () => {
       expect(result).to.deep.equal(expectedResult);
     });
   });
+
+  suite('getExportToLanguageMode', function () {
```

Comment:
Should we also add a test for error case?
What happens if this fails?

Reply:
Pushed updates for both of your comments! 🔥